c2rust-analyze/src/borrowck/type_check.rs

```diff
@@ -602,6 +602,9 @@ impl<'tcx> TypeChecker<'tcx, '_> {
                             self.visit_operand(p)
                         });
                     }
+                    Callee::Null { .. } => {
+                        // TODO: do we need to do anything here?
```

Comment:
I don't think we do. The only thing that comes to mind is calling `visit_place(destination)` as we do for other pointer-valued function calls, but if I'm understanding its definition properly, that call is actually pure and we don't need its result here.